webpack/components/Table/MainTable.js

```diff
@@ -13,24 +15,43 @@ import Loading from '../../components/Loading';
 const MainTable = ({
   status, cells, rows, error, emptyContentTitle, emptyContentBody,
   emptySearchTitle, emptySearchBody, searchIsActive, activeFilters,
-  ...extraTableProps
+  composable, rowsCount, children, ...extraTableProps
 }) => {
+  if (!composable && (!cells || !rows)) {
+    console.error(__('The <MainTable> component requires either a composable prop, or cells & rows props.')); // eslint-disable-line no-console
```

Comment:
I had to remove the `.isRequired` from `cells` and `rows` prop types, because they aren't required for composable tables.  So we're left needing to conditionally require props - either cells/rows, or `composable`.  (PropTypes does have a custom function, but it didn't really work for this purpose)

Reply:
#typescript ;)

Reply:
PropTypes is poor man's Typescript!  It's just as good :smile: